packages/mds-audit/types.ts

```diff
@@ -89,8 +89,8 @@ export interface AuditApiGetTripsRequest extends AuditApiRequest {
   >
 }
 
-export interface AuditApiGetTripRequest extends AuditApiTripRequest {
-  query: Partial<{ [P in 'event_viewport_adjustment']: string }>
+export type AuditApiGetTripRequest = AuditApiTripRequest & {
+  query: { [P in 'event_viewport_adjustment']: string }
```

Comment:
Can we create a generic type for this?
```ts
type ApiQuery<Q extends string> = { query: { [P in Q]: string } }
```

Reply:
For sure -- was avoiding that because I see this as a short term solution (with the folks at definitelytyped figuring out their express definitions, or us finding a solid workaround once there's bandwidth to tackle it), but happy to add

Reply:
Reminds me, this should be a `Partial` ApiQuery, I was changing methodology as I worked through things so there are still some lingering artifacts I need to fix up.